src/charts/line.js

```diff
@@ -787,12 +793,60 @@ define(function(require){
                 .enter()
                   .append('line')
                     .attr('class', 'extended-x-line')
-                    .attr('x1', (-xAxisPadding.left - 30))
+                    .attr('x1', (-xAxisPadding.left - verticalShift))
                     .attr('x2', chartWidth)
                     .attr('y1', height - margin.bottom - margin.top)
                     .attr('y2', height - margin.bottom - margin.top);
         }
 
+        /**
+         * Draws custom user-defined lines onto the chart
+         * @return void
+         */
+        function drawCustomLines(){
+            svg.select('.custom-lines-group')
+                .selectAll('line')
+                .remove();
+
+            let yValues = customLines.map(it => it.y);
```

Comment:
Nit: I would rename the item name from 'it' to 'line'

Reply:
Yeah, good point - it's much more readable. I'm used to Kotlin right now, there you would only write `.map(it.y)`.